### PR TITLE
doc: wifi: minor fixes in wifi docs

### PR DESCRIPTION
**Pull request by richabp**
Minor style fixes in Wi-Fi features guide and samples.

### CODE REVIEW

**Comment by richabp** [comment]
Issues in the QuickTrack sample are fixed in https://github.com/nrfconnect/sdk-nrf/pull/12444, hence not included in this PR.

File: samples/wifi/sr_coex/README.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ Wi-Fi: Bluetooth LE coexistence
    :local:
    :depth: 2
 
-This sample demonstrates coexistence between Wi-Fi® and Short Range (SR) radios in 2.4 GHz frequency.
+The Bluetooth LE coexistence sample demonstrates coexistence between Wi-Fi® and Short Range (SR) radios in 2.4 GHz frequency.
```
Comment (kittydepa) [inline samples/wifi/sr_coex/README.rst:10]:
```suggestion
The Bluetooth® Low Energy (LE) coexistence sample demonstrates coexistence between Wi-Fi® and Short Range (SR) radios in 2.4 GHz frequency.
```

Comment (kittydepa) [reply samples/wifi/sr_coex/README.rst:10]: Additionally, on line 23 the trademark for Bluetooth can be removed

**Review by kittydepa — CHANGES_REQUESTED** [review]

File: samples/wifi/sr_coex/README.rst (hunk 1 above)
Comment (richabp) [reply samples/wifi/sr_coex/README.rst:10]: I thought of this change but then I think this involves the Title change of the sample as well hence dropped it.

**Review by kittydepa — APPROVED** [review]

**Review by sachinthegreen — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Pull request merged**